etc/adminhtml/system.xml

```diff
@@ -49,6 +49,10 @@
                     <comment>This is the type of weight that I use with my products.</comment>
                     <source_model>MyParcelNL\Magento\Model\Source\WeightType</source_model>
                 </field>
+               <field id="retour_in_the_box" translate="label" type="select" sortOrder="216" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Retour in the box</label>
```

Comment:
Ik had hier als comment van Edie dat het misschien niet duidelijk is voor een klant dus ik heb:
"Print return label directly" misschien iets overeenkomends bedenken?